test/ui/pages/config/PageConfigPage.test.js

```diff
@@ -4,7 +4,7 @@ import 'test/enzyme-init';
 import PageConfigPage from 'ui/pages/config/PageConfigPage';
 import { shallowWithIntl } from 'test/testUtils';
 
-jest.unmock('react-redux');
+// jest.unmock('react-redux');
```

Comment:
please remove comments

Reply:
> very big work!
> there's a small comment to remove plus a question: why there are less tests now?

@gidesan most tests that are removed are related to table row tests which are mostly not applicable to the new table component. Though if needed, we can change the tests with the use of react testing library for easy tests but this will take a lot of time on producing unit tests.

Reply:
Yeah, please use RTL